Kommunicate/Classes/KMConversationService.swift

```diff
@@ -194,3 +200,56 @@ public class KMConversationService: KMConservationServiceable {
         return allBotIds
     }
 }
+
+func awayMessageFor(applicationKey: String, groupId: NSNumber, completion: @escaping (String?)->()) {
+    // Set up the URL request
+    let todoEndpoint: String = "https://api.kommunicate.io/applications/\(applicationKey)/awaymessage?conversationId=\(groupId)"
+    guard let url = URL(string: todoEndpoint) else {
+        print("Error: cannot create URL")
+        return
+    }
+    let urlRequest = URLRequest(url: url)
+
+    // set up the session
+    let config = URLSessionConfiguration.default
+    let session = URLSession(configuration: config)
+
+    // make the request
+    let task = session.dataTask(with: urlRequest) {
```

Comment:
We can move this network request to a separate method. 
Also, write test cases for all possibilities (like some error in json, or null data).